dask/dataframe/_compat.py

```diff
@@ -9,6 +9,7 @@
 PANDAS_GT_0240 = PANDAS_VERSION >= LooseVersion("0.24.0")
 PANDAS_GT_0250 = PANDAS_VERSION >= LooseVersion("0.25.0")
 PANDAS_GT_100 = PANDAS_VERSION >= LooseVersion("1.0.0")
+PANDAS_GT_104 = PANDAS_VERSION >= LooseVersion("1.0.4")
```

Comment:
Do we still need this?

Reply:
We're using `PANDAS_GT_104` over in `test_rolling_numba_engine` to know when to `xfail` the test